quill-core/src/main/scala/io/getquill/dsl/ValueComputation.scala

```diff
@@ -0,0 +1,139 @@
+package io.getquill.dsl
```

Comment:
I'd make a separate commit for such refactor before introducing returning record feature. So it'd be easy to track

Reply:
This makes sense. I'll try to separate it out.

Reply:
Okay. There are 4 commits now.